pom.xml

```diff
@@ -82,55 +83,58 @@
     <zipkin-reporter.version>2.15.2</zipkin-reporter.version>
 
     <!-- Ensure older versions of spring still work -->
-    <spring5.version>5.2.7.RELEASE</spring5.version>
+    <spring5.version>5.2.8.RELEASE</spring5.version>
     <spring.version>3.2.18.RELEASE</spring.version>
 
     <!-- Apis used, but not in Jetty 7.6* imply duplication in servlet25 test fixtures -->
     <!-- prefer sparkjava's jetty to reduce downloads -->
     <jetty.version>9.4.30.v20200611</jetty.version>
     <jetty-servlet25.version>7.6.21.v20160908</jetty-servlet25.version>
     <!-- Note: 3.1.x requires Java 8; 3.0.20.Final is broken -->
-    <resteasy.version>3.12.1.Final</resteasy.version>
+    <resteasy.version>3.13.1.Final</resteasy.version>
 
-    <kafka.version>2.5.0</kafka.version>
-    <!-- must align with kafka version https://github.com/charithe/kafka-junit -->
-    <kafka-junit.version>4.1.9</kafka-junit.version>
+    <kafka.version>2.6.0</kafka.version>
     <activemq.version>5.16.0</activemq.version>
-    <spring-rabbit.version>2.2.8.RELEASE</spring-rabbit.version>
+    <spring-rabbit.version>2.2.10.RELEASE</spring-rabbit.version>
 
-    <finagle.version>20.6.0</finagle.version>
+    <finagle.version>20.8.1</finagle.version>
     <log4j.version>2.13.3</log4j.version>
-    <okhttp.version>4.7.2</okhttp.version>
+    <okhttp.version>4.9.0</okhttp.version>
     <httpclient.version>4.5.12</httpclient.version>
 
-    <grpc.version>1.30.2</grpc.version>
+    <grpc.version>1.32.1</grpc.version>
+    <protobuf.version>3.12.0</protobuf.version>
     <!-- prefer grpc's version of netty -->
-    <netty.version>4.1.48.Final</netty.version>
+    <netty.version>4.1.51.Final</netty.version>
 
     <sparkjava.version>2.9.2</sparkjava.version>
+
+    <!-- Test only dependencies -->
     <junit.version>4.13</junit.version>
-    <assertj.version>3.16.1</assertj.version>
+    <assertj.version>3.17.2</assertj.version>
     <powermock.version>2.0.7</powermock.version>
-    <mockito.version>3.3.3</mockito.version>
+    <mockito.version>3.5.10</mockito.version>
     <jersey.version>2.31</jersey.version>
-    <jmh.version>1.23</jmh.version>
+    <!-- must align with kafka version https://github.com/charithe/kafka-junit -->
+    <kafka-junit.version>4.1.10</kafka-junit.version>
     <testcontainers.version>1.14.3</testcontainers.version>
 
+    <animal-sniffer-maven-plugin.version>1.19</animal-sniffer-maven-plugin.version>
```

Comment:
ordered lexicographical as tired of scanning

Reply:
thanks!! 🙏🏽